documentation/docs/steps/abapEnvironmentAssembly.md

```diff
@@ -0,0 +1,43 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+## Prerequisites
+* ???
```

Comment:
Before your step can be executed, the Communication Arrangement needs to be set up (similar to our steps). I would include it in the documentation

Reply:
Now added